exercises/practice/zebra-puzzle/.meta/example.cpp

```diff
@@ -0,0 +1,244 @@
+#include <algorithm>
+#include <iostream>
+#include <map>
+#include <numeric>
+#include <vector>
+
+using namespace std;
+
+namespace zebra_puzzle 
+{
+
+typedef struct 
+{
+    string drinksWater;
+    string ownsZebra;
+} Solution;
```

Comment:
If you '#include` the `.h` file you won't need to repeat this definition (see above).